sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractModuleConfiguration.java

```diff
@@ -47,9 +44,12 @@
  *
  * Note: even if the concept of "module" was dropped from the SQ server side,
  * "modules" are still a core concept of the SQ scanner.
+ *
+ * Module-independent functionality is in AbstractLanguageConfiguration.
```

Comment:
nitpick: use javadoc [see](https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#@see) `@see  package.class` for `AbstractLanguageConfiguration`

by `functionality` you mean rather `configuration`, right? because module-independent functionality is in many other classes

Reply:
I used `@link`. That's probably the intended behavior. `@see` generates the `See also` section and cannot be embedded into a line.